Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM praekeltfoundation/django-bootstrap:py3.6-stretch
+
+WORKDIR /casepro
+
+RUN apt-get-install.sh git wget curl && curl -sL https://deb.nodesource.com/setup_4.x | bash && apt-get install nodejs -y
```

Comment:
Maybe we should do the `apt-get-install.sh` for `wget` as part of the download of casepro? So that we can do an uninstall for `wget` afterwards, all in a single command, to reduce the size of the docker image?

Also, I'm not sure where we're using `git`.

We can probably install a newer version of `node`, probably the latest lts is good.

And then we can uninstall `curl` after we're done with it, in the same command.